scripts/make-calibration-ood.sh

```diff
@@ -4,5 +4,12 @@ set -e
 set -x
 
 test -f dropped-log.tsv || aws s3 cp s3://geniehai/gcampax/dropped-log.tsv dropped-log.tsv
-sed -E -e '/\t(ok|contextual command|ambiguous|redacted|meta-command|chatty)$/d' -e 's/\t/\tnull\t/' -e 's/\t[^\t]*$/\t\$failed ;/' dropped-log.tsv > calibration-ood.tsv
+
+# These are in-domain: ok|contextual command|ambiguous|redacted|meta-command|chatty
+# We ignore these: bug-tokenize
+# out of domain:
+sed -E -e '/\t(phone|foreign language|dialogue-model|gnome|math|new device|new function|policy or remote program|question|skill-help|web search|unintellegible|stream|tutorial)$/!d' -e 's/\t/\tnull\t/' -e 's/\t[^\t]*$/\t\$ood ;/' dropped-log.tsv > calibration-ood.tsv
```

Comment:
I would rather keep this as a negative list rather than a positive list. There might be more classes added in the future.

Also, chatty should be OOD. It was not included in the OOD set before because I need to reannotate some chatty sentences that are actually in-domain, but you should consider chatty as OOD.

Reply:
Then you will add those new classes to this list.
Seeing all the categories in this file helped me realize that `bug-tokenize` should be ignored for example, while you were considering it OOD.